$NetBSD: patch-aa,v 1.1 2011/01/10 11:40:17 obache Exp $

* The USE headers in DragonFly are located in bus/usb.

--- pluged.h.orig	2005-08-01 13:38:54.000000000 +0000
+++ pluged.h
@@ -1,8 +1,13 @@
 #ifndef __pluged_dat
 #define __pluged_dat
 
+#ifdef __DragonFly__
+#include <bus/usb/usb.h>
+#include <bus/usb/usbhid.h>
+#else
 #include <dev/usb/usb.h>
 #include <dev/usb/usbhid.h>
+#endif
 #include <sys/queue.h>
 #include <regex.h>
 
